src/java/org/apache/cassandra/index/accord/MemtableIndex.java

```diff
@@ -63,13 +65,23 @@ public Segment write(IndexDescriptor id) throws IOException
         return memoryIndex.write(id);
     }
 
-    public Collection<ByteBuffer> search(int storeId, TableId tableId, byte[] start, boolean startInclusive, byte[] end, boolean endInclusive)
+    public void search(int storeId, TableId tableId, byte[] start, byte[] end,
+                       Timestamp minTimestamp,
```

Comment:
why Timestamp vs TxnId? Would I think make intent clearer to stick to TxnId

Reply:
made this change, all tests are passing now